library/Zend/Form.php

```diff
@@ -1170,7 +1170,7 @@ public function addElements(array $elements)
                 } else {
                     switch ($argc) {
                         case 0:
-                            continue;
+                            continue 2;
```

Comment:
měl by tam být spíš break, to odpovídá původnímu chování (ono to není zásadní, protože ten switch je poslední ve foreach, ale dává to větší smysl)